pubspec.yaml

```diff
@@ -1,12 +1,15 @@
 name: logging
-version: 0.11.4-dev
-author: Dart Team <misc@dartlang.org>
+version: 0.11.3+2
```

Comment:
@kevmoo - I assumed that you wanted a version increase from `0.11.3+1` to `0.11.3+2` (rather than 0.11.4).

Reply:
Yup! I can explain later why I did the .4-dev thingy